### PR TITLE
Replace curl node interactions w/ websocat

### PR DESCRIPTION
**Pull request by alfarok**
### Description

Addresses https://github.com/w3f/polkadot-wiki/issues/3792

### CODE REVIEW

**Comment by alfarok** [comment]
Hey @NukeManDan, let me know if everything looks okay and we can get this merged.

File: docs/build/build-node-interaction.md [hunk 1]
```diff
@@ -42,9 +43,9 @@ Note that this call will show even those RPC methods which are disabled by a saf
 Add parameters in the call, for example get a block by its hash value:
 
 ```bash
-$ curl -H "Content-Type: application/json" -d '{"id":1, "jsonrpc":"2.0", "method": "chain_getBlock", "params":["0x3fa6a530850324391fde50bdf0094bdc17ee17ec84aca389b4047ef54fea0037"]}' http://localhost:9933
+echo '{"id":1,"jsonrpc":"2.0","method":"chain_getBlock","params":["0x7d4ef171d483d37aa2339877524f0731af98e367c38f8fa27f133193ed2b5615"]}' | websocat -n1 -B 99999999 ws://127.0.0.1:9944
```
Comment (alfarok) [inline docs/build/build-node-interaction.md:46]: I changed this hash to another random block because the original didn't seem to exist.

Comment (nuke-web3) [reply docs/build/build-node-interaction.md:46]: this is likely from your node still sinking, or pruning. Using polkadot (I assume this is your network?) your example on my local node also fails to get this while syncing. 

Works fine with  `wss://rpc.polkadot.io` as the endpoint.

Comment (alfarok) [reply docs/build/build-node-interaction.md:46]: @NukeManDan strange, I am also using `wss://rpc.polkadot.io` to get results locally.  When I run the command using the original address hash:
```bash
echo '{"id":1,"jsonrpc":"2.0","method":"chain_getBlock","params":["0x3fa6a530850324391fde50bdf0094bdc17ee17ec84aca389b4047ef54fea0037"]}' | websocat -n1 -B 99999999 wss://rpc.polkadot.io
```

I get a null response:
```json
{"jsonrpc":"2.0","result":null,"id":1}
```

But running with the other random hash:
```bash
echo '{"id":1,"jsonrpc":"2.0","method":"chain_getBlock","params":["0x7d4ef171d483d37aa2339877524f0731af98e367c38f8fa27f133193ed2b5615"]}' | websocat -n1 -B 99999999 wss://rpc.polkadot.io
```

I get a valid response with a large collection of `logs` data:
```json
{"jsonrpc":"2.0","result":{"block":{"header":{"parentHash":"0xb5e10293122a3c706dfcf5c0e89d5fb90929e7ee580c5167e439afa330fae2c7","number":"0xbb07fe","stateRoot":"0x872dfbb3516a6e3b9becf01bb2192e53a1d77ef6c37e426f03ebf64b33a68ede","extrinsicsRoot":"0xe131e6af57c503ca6c6a151b2e621d05f65ef7be07e24abc2444fa1eb67c444a","digest":{"logs":[...]},"justifications":null},"id":1}
```

Comment (alfarok) [reply docs/build/build-node-interaction.md:46]: I don't see any results on subscan either for that hash

Comment (nuke-web3) [reply docs/build/build-node-interaction.md:46]: My bad, the new hash us what i was referring working, only after sync. Your chage is great 🙌

**Pull request merged**